Dockerfile-builder

```diff
@@ -28,3 +20,8 @@ RUN npm install -g \
     jshint@2.7.0 \
     stylus@0.49.2 \
     uglify-js@2.4.13
+
+COPY ./requirements/compiled.txt /app/requirements/compiled.txt
+
+RUN pip install --build ./build --no-cache-dir --no-deps \
+    -r requirements/compiled.txt
```

Comment:
This seems like it would still be suitable for using as a local-dev Dockerfile. We could just point the docker-compose `dockerfile` to this. Is that correct? Or would we have another Dockerfile for local dev?